benchmarks/build.gradle.kts

```diff
@@ -130,3 +134,5 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
 readme {
     maturity = ru.mipt.npm.gradle.Maturity.EXPERIMENTAL
 }
+
+addBenchmarkProperties()
```

Comment:
Is it possible to make it an extension of https://github.com/mipt-npm/gradle-tools/blob/670a4ca71ee0a9b714eaef63fa7582ad478fedf9/src/main/kotlin/ru/mipt/npm/gradle/KScienceReadmeExtension.kt#L17? Not critical though.

Reply:
This function highly depends on `benchmarks` `Project`

Reply:
ReadmeExtension has access to project via its `project` field.